### PR TITLE
Add STS rule overview with dependencies

### PR DESCRIPTION
**Pull request by mgudemann**

### CODE REVIEW

File: shelley/chain-and-ledger/formal-spec/ledger-spec.tex [hunk 1]
```diff
@@ -223,6 +223,7 @@
 \include{ledger}
 \include{epoch}
 \include{chain}
+\include{sts-overview}
```
Comment (JaredCorduan) [inline shelley/chain-and-ledger/formal-spec/ledger-spec.tex]: did you mean to add a `sts-overview.tex` file that displayed the new image?

Comment (mgudemann) [reply shelley/chain-and-ledger/formal-spec/ledger-spec.tex]: yes, sorry, forgot that

**Comment by dnadales** [comment]
Note that we could get these for "free" (as in free shrink :P) if we structure our rules in modules and use `graphmod`, e.g.:

image

For Byron I added [this task](https://github.com/input-output-hk/cardano-ledger/wiki/Roadmap#reorganize-the-byron-ledger-rules) to the "work proposals" section of our roadmap.

**Comment by mgudemann** [comment]
@dnadales nice, didn't know that. But I also wanted to have the consumer / producer dependencies of the sub-rules, would that be possible, too ?

**Comment by dnadales** [comment]
> @dnadales nice, didn't know that. But I also wanted to have the consumer / producer dependencies of the sub-rules, would that be possible, too ?

That much I don't know but isn't the direction of the arrow enough?

**Comment by dnadales** [comment]
Oh, I see. Then no, I don't think we'll get that for free :)

**Pull request merged**